DrawioEditor.php

```diff
@@ -24,6 +24,8 @@
 /* Config Defaults */
 $wgDrawioEditorImageType = 'svg';
 $wgDrawioEditorImageInteractive = false;
+$wgDrawioEditorUrl = "https://www.draw.io";
+$wgDrawioEditorLocal = false;
```

Comment:
Why are we creating a dedicated configuration here? Couldn't we just assume that if it is not the original service URL, it's a local URL?

E.g. `var localAttr = this.url !== 'https://www.draw.io' ? "&local=1" : "";`

Reply:
fixed